docs/requirements.txt

```diff
@@ -1,13 +1,17 @@
 absl-py==0.10.0
 alabaster==0.7.12
+alembic==1.4.1
```

Comment:
Do we need to update those packages in order to fix the readthedocs build issue? How about we just add `image: stable`

Reply:
- As per docs/readme, we should have new lib added in requirement.txt, I was trying to add Mlflow dependency so added through given command "poetry export --dev --without-hashes -f requirements.txt > docs/requirements.txt". It added alot other dependency other than mlflow.
- Let me try building without changing dependency.

Reply:
Ran 4 build only on stable image, 2 passed 2 failed, not sure whether stable image will help or not.